### PR TITLE
[tools/quake3/q3map2] Fix memset size bugs

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

**Comment by Garux** [comment]
i think correct one must be
```
		memset( verts, 0, sizeof( *verts ) * maxSurfaceVerts );
		memset( indexes, 0, sizeof( *indexes ) * maxSurfaceIndexes );
```
in surface_meta.c
this made on effect on resulting bsp in tests though
perhaps, zeroing 1st byte is enough here (and is slightly quicker)

**Comment by tkoeppe** [comment]
@Garux: Yes, you're right, the allocation was for multiple items. I fixed it and repushed.

(By the way, you know you can make comments in the code so they appear directly at the affected line?)

**Comment by Garux** [comment]
Yes, thanks.

File: tools/quake3/q3map2/bspfile_abstract.c [hunk 1]
```diff
@@ -113,7 +113,7 @@ void SetDrawSurfacesBuffer(){
 
 	bspDrawSurfaces = safe_malloc_info( sizeof( bspDrawSurface_t ) * numBSPDrawSurfacesBuffer, "IncDrawSurfaces" );
 
-	memset( bspDrawSurfaces, 0, MAX_MAP_DRAW_SURFS * sizeof( bspDrawVert_t ) );
+	memset( bspDrawSurfaces, 0, MAX_MAP_DRAW_SURFS * sizeof( bspDrawSurface_t ) );
```
Comment (TTimo) [inline tools/quake3/q3map2/bspfile_abstract.c]: Needs to use numBSPDrawSurfacesBuffer

**Pull request merged**